cabal-install/Distribution/Client/CmdInstall.hs

```diff
@@ -678,10 +678,10 @@ warnIfNoExes :: Verbosity -> ProjectBuildContext -> IO ()
 warnIfNoExes verbosity buildCtx =
   when noExes $
     warn verbosity $
-    "You asked to install executables, but there are no executables in "
+    "\x1b[31m You asked to install executables, but there are no executables in "
```

Comment:
If we go for colors, we should do it properly by at least checking whether terminal supports them.

Reply:
FWIW, also it's a warning. Not an error. Warnings are magenta (like in GHC :) )

Reply:
> If we go for colors, we should do it properly by at least checking whether terminal supports them.

Yeah, i guess in win dos only we'll see the funny escape codes. Well maybe that would attract the eye of users too 😄

Reply:
According to the internet ansi escape codes work in modern windows terminals.  I have no way to check.  Regardless, this doesn't mean the current terminal has support, I'd just expect most do.